packages/aws-lambda-otel-extension/opt/otel-extension/external/lambda-apis/runtime-api.js

```diff
@@ -24,6 +24,22 @@ async function getEventData() {
   };
 }
 
+async function getRuntimeEventData() {
+  const res = await fetch(`${baseUrl}/invocation/next`, {
```

Comment:
If I see correctly this is a Lambda runtime (not extension) API call.

Why exactly to we fall into that? Are there no implication on relying on it in lambda extensions ? (afaik it's dedicated for custom runtime implementations not extension).

Reply:
I don't think we need to use the runtime API for the `requestId`, it looks like we can get the `requestId` from the extension API call. I'll test it and update this if it works since Dan is out for the week.

Reply:
This should be fixed now to use the `requestId` that comes from the extension API